File: app/src/main/kotlin/com/krsk/qiitareader/data/repository/datasource/item/ItemDataStore.java
package com.krsk.qiitareader.data.repository.datasource.item;

import com.krsk.qiitareader.domain.model.entity.Item;

import java.util.List;

import rx.Observable;

/**
 * Created by bl-lia on 2/1/16.
 */
public interface ItemDataStore {

    Observable<List<Item>> getItems();
}

File: app/src/main/kotlin/com/krsk/qiitareader/data/repository/datasource/item/ItemDataStore.kt
package com.krsk.qiitareader.data.repository.datasource.item

import com.krsk.qiitareader.domain.model.entity.Item

import rx.Observable

/**
 * Created by bl-lia on 2/1/16.
 */
interface ItemDataStore {
    fun getItems(): Observable<List<Item>>
}
